docs/source/behav_at_swc/index.md

```diff
@@ -1,6 +1,6 @@
 # Behaviour at the SWC
```

Comment:
Not part of this PR, but I think we should remove the "at the SWC" bits. It gets repetitive, and the homepage states:
> The information is primarily aimed at researchers at the [Sainsbury Wellcome Centre (SWC)](https://www.sainsburywellcome.org/web/) and [Gatsby Computational Neuroscience Unit (GCNU)](https://www.ucl.ac.uk/gatsby/gatsby-computational-neuroscience-unit)

```suggestion
# Behaviour
```